src/Microsoft.AspNetCore.Razor.TagHelperTool/Application.cs

```diff
@@ -20,8 +20,9 @@ public Application()
 
             HelpOption("-?|-h|--help");
 
-            TagHelperManifest = Option("-o", "output file", CommandOptionType.SingleValue);
             Assemblies = Argument("assemblies", "assemblies to search for tag helpers", multipleValues: true);
+            TagHelperManifest = Option("-o", "output file", CommandOptionType.SingleValue);
+            ProjectRoot = Option("-p", "project root directory", CommandOptionType.SingleValue);
```

Comment:
I understand we didn't want to change this. But this looked necessary as we don't always want to rely on the CurrentDirectory to be the our working directory.

Reply:
Agreed. We can't rely on `Environment.CurrentDirectory` unless we know we're running as a CLI